docker-local/auth-service/spec_files/rules_response/default.json

```diff
@@ -196,5 +176,99 @@
     },
     "type": "response",
     "version": 1
+  },
```

Comment:
duplicate rule of the one you edited above?

Reply:
Strange... I think it automatically added it perhaps.

Reply:
If you're editing the both in your editor and on `testing-toolkit-ui` I believe that causes duplication in some occurances.